### ISSUE TITLE
新增其他语言

### ISSUE THREAD

**Issue by mustapa0** [fishaudio/fish-speech#383]
我想新增其他语言训练，但是出现了合成以后的音频无法理解的问题。
现在是否能添加其他类型的语言？

**Comment by yy524** [comment]
您好，我现在也尝试使用其他语言训练，也是合成声音不可懂。我是使用2000h数据finetune的。请问您现在有进展吗？

**Comment by mustapa0** [comment]
你用的其他语言是那个语言，可以用llm微调试试

**Comment by mustapa0** [comment]
> 您好，我现在也尝试使用其他语言训练，也是合成声音不可懂。我是使用2000h数据finetune的。请问您现在有进展吗？

你用的其他语言指那个？你可以用llm微调试试。

**Comment by jizh18** [comment]
你们好，我们也在用其他语言尝试训练，我们是面向卢干达语做训练，你们的训练集都多大呀，参数怎么设置的

**Comment by yy524** [comment]
> > 您好，我现在也尝试使用其他语言训练，也是合成声音不可懂。我是使用2000h数据finetune的。请问您现在有进展吗？
> 
> 你用的其他语言指那个？你可以用llm微调试试。

我是使用泰语训练的，llama的finetune和pretrain都尝试过，但是loss都不正常，合成声音也不正常。请问您那边现在声音正常了吗？是finetune的llama？

**Comment by jizh18** [comment]
> > > 您好，我现在也尝试使用其他语言训练，也是合成声音不可懂。我是使用2000h数据finetune的。请问您现在有进展吗？
> > 
> > 
> > 你用的其他语言指那个？你可以用llm微调试试。
> 
> 我是使用泰语训练的，llama的finetune和pretrain都尝试过，但是loss都不正常，合成声音也不正常。请问您那边现在声音正常了吗？是finetune的llama？

确实 兄弟 我是用卢干达语训练的 loss的变化也不太正常 另外我遇到一个问题 不使用lora的条件下 得到的模型在使用时在generate声音步骤会出现报错 你有遇到这个问题吗

**Comment by mustapa0** [comment]
> > > > 您好，我现在也尝试使用其他语言训练，也是合成声音不可懂。我是使用2000h数据finetune的。请问您现在有进展吗？
> > > 
> > > 
> > > 你用的其他语言指那个？你可以用llm微调试试。
> > 
> > 
> > 我是使用泰语训练的，llama的finetune和pretrain都尝试过，但是loss都不正常，合成声音也不正常。请问您那边现在声音正常了吗？是finetune的llama？
> 
> 确实 兄弟 我是用卢干达语训练的 loss的变化也不太正常 另外我遇到一个问题 不使用lora的条件下 得到的模型在使用时在generate声音步骤会出现报错 你有遇到这个问题吗

你的音频长度是多少？是否存在1～2秒这种类似的音频？

**Comment by Stardust-minus** [comment]
https://github.com/fishaudio/fish-speech/issues/515 Change title to English.

**Comment by Marysaadjousef** [comment]
Hello, I have a question about your experience training the model on a new language.
How long did the training process take, and what GPU(s) did you use?

**Comment by SmartManoj** [comment]
Tamil language support.

**Comment by abhisirka2001** [comment]
Has anyone got good results on finetuning the model in any language?

**Comment by abhisirka2001** [comment]
> Tamil language support.

Hey @SmartManoj 
Could you make any progress in finetuining for indic language?
Needed some help as i am trying for hindi language lora finetuning.

**Comment by SmartManoj** [comment]
I haven't fine-tuned yet. You need to create a new issue regarding that.

**Comment by kapilkarda** [comment]
Let me know when indic language support will be done?

**Comment by SmartManoj** [comment]
Reason for closing?
